code/modules/soulcrypt/access_update_tool.dm

```diff
@@ -55,7 +55,8 @@
 			return
 		//Else, we overwrite the soulcrypt's current access with the access on the ID card.
 		crypt.access.Cut()
-		crypt.access = card.access
+		for(var/x in card.access)
+			crypt.access += x
```

Comment:
An alternative would be to do [crypt.Copy()](http://www.byond.com/docs/guide/chap10.html), though that may require length arguments. Either way, this works to de-reference the list, assuming the vars in access are not also references.

Reply:
Copy() did not work when attempted. It requires some conversion to Lazylen first from what I can tell.